### PR TITLE
add css buttons #151

### PR DESCRIPTION
**Pull request by proseLA**
hi cindy,
i've finally decided to update my clients on this plugin.  **GREAT** job on the math....  i think...  i had some issues with the display on some tax rates, but my client uses something called OT_LOCAL_TAXES; i'll look at it further at some point.  but the numbers added up perfectly and were correct!  bravo!

as i talked about in february, i'm doing some updates to the look of this plugin.  i'm not sure if you want them or not.  you can let me know.  i added a couple of methods.  1 was stolen directly from the latest orders script of v157.  perhaps we can get them to put that into a class or function and not have to duplicate it here.

please let me know if you want this stuff.  i'm trying to stick to the look in this branch.

best,
pRose.

### CODE REVIEW

**Comment by lat9** [comment]
Thanks for that.  I'll hold it until such time as EO no longer supports the zc155 series (which doesn't support bootstrap).

**Comment by proseLA** [comment]
cindy,
i think this will handle v155 and up...

you definitely want to review it.   no logic change, i hope...  

best.

File: YOUR_ADMIN/edit_orders.php [hunk 1]
```diff
@@ -1977,7 +2012,17 @@
...
 
                     <tr>
-                        <td valign="top"><input type="submit" class="btn btn-danger" value="<?= IMAGE_UPDATE; ?>" ></td>
+                        <td valign="top">
+                            <?php
+                            if ((PROJECT_VERSION_MAJOR . '.' . PROJECT_VERSION_MINOR) < '1.5.6') {
+                                echo zen_image_submit('button_update.gif', IMAGE_UPDATE);
+                            } else {
+                                ?>
+                                <input type="submit" class="btn btn-danger" value="<?= IMAGE_UPDATE; ?>">
```
Comment (lat9) [inline YOUR_ADMIN/edit_orders.php:2021]: Full `<?php echo`, please.

File: YOUR_ADMIN/edit_orders.php [hunk 2]
```diff
@@ -1139,9 +1167,13 @@
 
     $additional_inputs = '';
     $zco_notifier->notify('EDIT_ORDERS_FORM_ADDITIONAL_INPUTS', $order, $additional_inputs);
+    if  ((PROJECT_VERSION_MAJOR . '.' . PROJECT_VERSION_MINOR) >= '1.5.6') {
     ?>
                             <input type="submit" class="btn btn-danger" value="<?= IMAGE_UPDATE; ?>" >
                         <?= "&nbsp;$reset_totals_block&nbsp;$payment_calc_choice$additional_inputs";
```
Comment (lat9) [inline YOUR_ADMIN/edit_orders.php:1173]: Full `<?php echo`, please.

File: YOUR_ADMIN/includes/classes/editOrders.php [hunk 3]
```diff
@@ -755,12 +780,25 @@ function orderNavigation($oID)
...
-        <a href="<?php echo zen_href_link(FILENAME_ORDERS, zen_get_all_get_params(array('action'))); ?>"
-           class='btn btn-primary btn-sm' role='button'><?= IMAGE_BACK; ?></a>
-        <a href="<?php echo zen_href_link(FILENAME_ORDERS,
-            zen_get_all_get_params(array('oID', 'action')) . "oID=$oID&amp;action=edit"); ?>"
-           class='btn btn-primary btn-sm' role='button'><?= DETAILS; ?></a>
-        <?php
+        if ((PROJECT_VERSION_MAJOR . '.' . PROJECT_VERSION_MINOR) > '1.5.5') {
+            ?>
+            <a href="<?php echo zen_href_link(FILENAME_ORDERS, zen_get_all_get_params(array('action'))); ?>"
+               class='btn btn-primary btn-sm' role='button'><?= IMAGE_BACK; ?></a>
```
Comment (lat9) [inline YOUR_ADMIN/includes/classes/editOrders.php:786]: Full `<?php echo`, please.

File: YOUR_ADMIN/includes/classes/editOrders.php [hunk 4]
```diff
@@ -755,12 +780,25 @@ function orderNavigation($oID)
...
+            <a href="<?php echo zen_href_link(FILENAME_ORDERS,
+                zen_get_all_get_params(array('oID', 'action')) . "oID=$oID&amp;action=edit"); ?>"
+               class='btn btn-primary btn-sm' role='button'><?= DETAILS; ?></a>
```
Comment (lat9) [inline YOUR_ADMIN/includes/classes/editOrders.php:789]: Ditto.

File: YOUR_ADMIN/edit_orders.php [hunk 5]
```diff
@@ -1984,7 +1995,7 @@ function init()
                     </tr>
 
                     <tr>
-                        <td valign="top"><?php echo zen_image_submit('button_update.gif', IMAGE_UPDATE); ?></td>
+                        <td valign="top"><input type="submit" class="btn btn-danger" value="<?= IMAGE_UPDATE; ?>" ></td>
```
Comment (lat9) [inline YOUR_ADMIN/edit_orders.php]: Ditto.

File: YOUR_ADMIN/includes/classes/editOrders.php [hunk 6]
```diff
@@ -689,4 +689,78 @@ public function eoGetOrderTotalSortOrder($order_total_code)
...
+    }
+
+    function backDetails($oID)
+    {
+        ?>
+        <a href="<?php echo zen_href_link(FILENAME_ORDERS, zen_get_all_get_params(array('action'))); ?>"
+           class='btn btn-primary btn-sm' role='button'><?= IMAGE_BACK; ?></a>
+        <a href="<?php echo zen_href_link(FILENAME_ORDERS,
+            zen_get_all_get_params(array('oID', 'action')) . "oID=$oID&amp;action=edit"); ?>"
+           class='btn btn-primary btn-sm' role='button'><?= DETAILS; ?></a>
```
Comment (lat9) [inline YOUR_ADMIN/includes/classes/editOrders.php]: Ditto.

File: YOUR_ADMIN/includes/classes/editOrders.php [hunk 7]
```diff
@@ -689,4 +689,78 @@ public function eoGetOrderTotalSortOrder($order_total_code)
...
+            </div>
+        </div>
+        <?php
+    }
+
+    function backDetails($oID)
+    {
+        ?>
+        <a href="<?php echo zen_href_link(FILENAME_ORDERS, zen_get_all_get_params(array('action'))); ?>"
+           class='btn btn-primary btn-sm' role='button'><?= IMAGE_BACK; ?></a>
```
Comment (lat9) [inline YOUR_ADMIN/includes/classes/editOrders.php]: Ditto.

**Comment by proseLA** [comment]
totally acceptable in psr-1 coding standard.  and much cleaner looking, IMO.

https://www.php-fig.org/psr/psr-1/

but your repo....   :smirk:

**Pull request merged**